src/platform/posix/posix_ipcconn.c

```diff
@@ -33,6 +33,12 @@
 #define MSG_NOSIGNAL 0
 #endif
 
+#ifndef SOL_LOCAL
+#ifdef __FreeBSD__
```

Comment:
I don't think you need the __FreeBSD__ guard here.  I'm fairly confident that the systems in question either already have it defined, or use 0 (and not some other value.)